core/src/components/modal/test/canDismiss/e2e.ts

```diff
@@ -0,0 +1,81 @@
+import { newE2EPage } from '@stencil/core/testing';
+import { dragElementBy } from '@utils/test';
+
+describe('modal - canDismiss handler', () => {
+  let page;
```

Comment:
```suggestion
  let page: E2EPage;
```

& add `E2EPage` to the import for `@stencil/core/testing`.